jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -500,6 +575,13 @@ boolean tryRelease()
             return !(overUsed && newMultiplexingCount == 0);
         }
 
+        public boolean isOverUsed()
```

Comment:
This doesn't seem to be used.

Reply:
I did have a use for it.... but can't remember now... was probably some esoteric strategy or perhaps a unit test.
I still think it is a nice to have along with the `isIdle` type methods.

Reply:
+1 for keeping the method for JMX and such.